src/FSharp.Core/reflect.fs

```diff
@@ -1169,18 +1216,13 @@ type FSharpType =
     static member MakeTupleType(types: Type[]) =
         checkNonNull "types" types
 
-        // No assembly passed therefore just get framework local version of Tuple
```

Comment:
Will this ever be called from a build doing cross targetting, I.e a desktop compiler building a coreclr application or a coreclr FSC building a desktop application.  Perhaps as part of a type provider?

If so is this a correct assumption?   Was it ever a correct assumption?

Reply:
It behaves same as before.
Only before, the call to `` typeof<System.Tuple>.Assembly `` happened here in the outer MakeTupleType call.

After the change, it is implicit inside mkTupleTypeNetStandard, because it works with an array of types `` refTupleTypesNetStandard `` created the same mechanism.